### DISCUSSION TITLE
Mail alert (Alert, hunt, fleet, wazuh...)

### DISCUSSION THREAD

**Discussion by DashCarlos** [Security-Onion-Solutions/securityonion#2062] [Unsupported Versions] [answered]
Hello! Before starting, I appreciate your help and patience. I would like to configure the mail alerts for the Suricata module mainly, I have seen that I can also do it for Wazuh that has its own implementation but I want to focus on Suricata.
I understand that to configure the mailing I have to modify the ElastAlert configuration, is this true?
In addition, there is something that I do not fully understand, Salt configures the services included in SecuriyOnion, for many of the configuration files there are the same files that hang from different directories, if for example I want to modify the ElastAlert configuration, what file should I modify? I'm concerned about this because I don't want SecurityOnion updates using Salt override my settings. Thank you!
I'm using version 2.3.10

**Comment by weslambert** [comment, marked as answer]
You should be able to place your custom Elastalert rules in `/opt/so/rules/elastalert/` (with your email configuration, and they should not be modified or removed.
